packages/legend-application-query/src/stores/QueryEditorStore.ts

```diff
@@ -379,6 +383,30 @@ export abstract class QueryEditorStore {
       query.content =
         await this.graphManagerState.graphManager.lambdaToPureCode(rawLambda);
       config?.decorator?.(query);
+      // any error in default parameters we will ignore and only log
+      query.defaultParameterValues = undefined;
```

Comment:
The biggest concern I have here is, should we make this implied default behavior or this should be `a checkbox` when users save/saveAs their queries. What do you think?

Reply:
i think for now implied behaviour. we can look to maybe add it to the save modals but for me it should be like a hidden feature for now.

Reply:
TODO: as dicussed, I think we should have something like a checkbox in the save/saveas modal to let people decide if they want to persist the parameters' values as default values or not